data/expected_match.csv

```diff
@@ -0,0 +1,7 @@
+email_address_field,1,3
+participant1@bhg.org,0.6666666666666666,0.21428571428571427
```

Comment:
I think we should limit the output to 2 decimals.

Reply:
Done in b5912c7.